zones/trollpasta.com

```diff
@@ -30,3 +30,4 @@ www		DYNA	geoip!cp
 ; load balancers
 
 ; Other
+trollpasta.com.	TXT	google-site-verification=t0_E9O7AEVjARLYxq1d6hfjd6OQzzo5bVyLELdRmOKI.
```

Comment:
the text needs to be in quotation marks "" and there wouldn't be a period at the end.